heareval/predictions/task_predictions.py

```diff
@@ -672,10 +684,12 @@ def dataloader_from_split_name(
     return DataLoader(
         dataset,
         batch_size=batch_size,
-        # We don't shuffle because it's slow.
+        # We don't shuffle because it's slow
+        # (except when in_memory = True).
         # Also we want predicted labels in the same order as
-        # target labels.
+        # target labels, for validation and test.
         shuffle=False,
+        pin_memory=True,
```

Comment:
Is there any consideration with the metadata? With this I think the embeddings and labels will be transferred to CUDA, but the metadata won't (I think b/c they aren't tensors). I think it will be fine, just curious if there are any gotchas there.

Reply:
TBH I don't know 

https://github.com/neuralaudio/hear-eval-kit/issues/294

https://github.com/PyTorchLightning/pytorch-lightning/discussions/9340